src/shared/components/FormComponents/EditorController/EditorController.tsx

```diff
@@ -19,53 +19,68 @@ export const EditorController = <T extends FieldValues>({
   withDebounce = false,
   'data-testid': dataTestid,
 }: EditorControllerProps<T>) => {
+  const dispatch = useAppDispatch();
   const editorRef = useRef<ExposeParam>();
-  const [fileSizeExceeded, setFileSizeExceeded] = useState<number | null>(null);
-  const [incorrectFileFormat, setIncorrectFileFormat] = useState<MediaType | null>(null);
+
+  const onFileSizeExceeded = useCallback(
```

Comment:
I'm curious, what are the benefits of using useCallback here and in the next function?

Reply:
I guess to not re-render the huge component `<Editor` (use the old reference in properties when parent component is re-rendered). 
I think it makes sense, if it has a performance issues. Otherwise, I would consider it as a overengineering.